extide/gradle/src/org/netbeans/modules/gradle/loaders/LegacyProjectLoader.java

```diff
@@ -420,10 +454,30 @@ private static GradleReport createReport(Throwable e) {
         } else {
             reported = e;
         }
+        String cn = e.getClass().getName();
+        if (cn.contains("GradleScriptException") || cn.contains("ResolutionException")) {
```

Comment:
These are common exception classes for failures reported by the build system itself, mark as 'user' or expected exceptions.